### PR TITLE
mod_mam_odbc_async_pool_writer and mod_mam_muc_odbc_async_pool_writer configurable pool size

### PR DESCRIPTION
**Pull request by vooolll**
solving this task https://github.com/esl/MongooseIM/issues/395

### CODE REVIEW

**Comment by vooolll** [comment]
====== Test name: test_kill

====== Reason: {{badmatch,[]},

[{system_monitor_SUITE,

test_kill_consume_large_heap_message,2,

[{file,"system_monitor_SUITE.erl"},{line,138}]},

{system_monitor_SUITE,test_kill_wait_for_ok,3,

[{file,"system_monitor_SUITE.erl"},{line,126}]},

{escalus_story,story,3,

[{file,"src/escalus_story.erl"},{line,39}]},

{test_server,ts_tc,3,

[{file,"test_server.erl"},{line,1360}]},

{test_server,run_test_case_eval1,6,

[{file,"test_server.erl"},{line,977}]},

{test_server,run_test_case_eval,9,

[{file,"test_server.erl"},{line,925}]}]}

Cant't pass this test, any idea what is wrong?

**Comment by michalwski** [comment]
This happens sometime when travis is overloaded. The job was restarted and this time it passed.

File: apps/ejabberd/src/mod_mam_muc_odbc_async_pool_writer.erl [hunk 1]
```diff
@@ -61,7 +63,7 @@ worker_prefix() ->
 %% or
 %% `worker_count(_) = 16, partition_count() = 16'.
 worker_count(_Host) ->
```
Comment (lavrin) [inline apps/ejabberd/src/mod_mam_muc_odbc_async_pool_writer.erl]: A leading underscore means "unused variable" and disables the compiler warning about that. Please don't use variables which are marked as unused / ignored, instead remove the underscore.

Comment (vooolll) [inline apps/ejabberd/src/mod_mam_muc_odbc_async_pool_writer.erl]: I fixed it in my new commit :+1:

**Comment by lavrin** [comment]
Looks good! Thanks for you effort.

**Pull request merged**